rewrite-core/src/main/java/org/openrewrite/internal/StringUtils.java

```diff
@@ -720,4 +720,19 @@ public static String formatUriForPropertiesFile(String uri) {
     public static boolean hasLineBreak(@Nullable String s) {
         return s != null && LINE_BREAK.matcher(s).find();
     }
+
+    public static String substringOfBeforeFirstLineBreak(String s) {
+        String[] lines = LINE_BREAK.split(s);
+        return lines.length > 0 ? lines[0] : "";
+    }
+
+    public static String substringOfAfterFirstLineBreak(String s) {
+        String[] lines = LINE_BREAK.split(s);
+        return lines.length > 1 ? String.join("\n", Arrays.copyOfRange(lines, 1, lines.length)) : "";
+    }
+
+    public static String substringOfAfterLastLineBreak(String s) {
+        String[] parts = LINE_BREAK.split(s);
+        return parts.length > 1 ? parts[parts.length - 1] : "";
+    }
```

Comment:
Let's not yet pull up these single use methods to the more widely used StringUtils. Even if it's internal, it's still public and exposed, whereas these can be private to their single usage site.

Reply:
👍🏻, I removed them!